pkg/storage/s3.go

```diff
@@ -338,6 +338,7 @@ func (rs *S3Storage) ReadFile(ctx context.Context, file string) ([]byte, error)
 
 	result, err := rs.svc.GetObjectWithContext(ctx, input)
 	if err != nil {
+		log.Error("In read s3 file", zap.Any("file info", input))
 		return nil, errors.Trace(err)
```

Comment:
I think it's better to wrap this info in error. such as `	return nil, errors.Annotatef(err,"failed to read s3 file in bucket %s %s.", input.Bucket, input.Key)
`

Reply:
@3pointer OK, I will make a change